JackyZhang1219/books-and-breakfast-app
# added edit feature for school transportation pages and added tracker link

### Description

I made an edit feature for the school transportation pages. The edit view/mode can be toggled by clicking the fixed edit icon button on the bottom right of the screen and you can click anywhere outside of the text boxes or on the icon again to save the changes. 

### QA

Please explain in detailed steps how the reviewer can test your changes and what the expected behaivors are (visual would help!). Consider different test cases/scenarios.

When you are on longer article pages like Willard Elementary, the keyboard blocks the bottom textbox from view (which we don't want). The edit icon should also stay fixed on the screen on the bottom right even when you scroll.
 
### Packages

Please describe any additional packages you added to package.json.

Material icons from react native vector icons

### Checklist
